_readings/sister-outsider.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: sister-outsider
+title: "Sister Outsider"
+authors: "Audre Lorde"
+categories: race
+goodreads_url: https://www.goodreads.com/book/show/32951.Sister_Outsider
+amazon_url: https://www.amazon.com/Sister-Outsider-Speeches-Crossing-Feminist-ebook/dp/B006L7RCEI/
```

Comment:
Also @pamo I would love your opinion on whether property names like these should use underscores or hyphens. I'm tempted to make them all hyphens but I can't find any clear style guides online.